telebot/types.py

```diff
@@ -7703,7 +7788,7 @@ def de_json(cls, json_string):
         obj = cls.check_json(json_string)
         return cls(**obj)
 
-    def __init__(self, request_id: int, chat_id: int) -> None:
+    def __init__(self, request_id: int, chat_id: int, **kwargs) -> None:
```

Comment:
What is the purpose of kwargs when we don't use it?

Reply:
> What is the purpose of kwargs when we don't use it?

IMO **kwargs saves library from updates of API. If there appears new fields - they will be autoparsed and passed to init. If there will be no dummy **kwargs to accept them - init call will fail with incorrect number of aguments. Isn't it?

We (you :) ) created these **kwargs long ago when invented this new method of calling init: I just allign some methods to the same signature as others.